### ISSUE TITLE
Deploy skeleton on staging server

### ISSUE THREAD

**Issue by strazdinsg** [The-Compilers/CryptoYard#20]
User story: As a developer I can see the skeleton-solution (Created in #4) running on the staging server. A frontend endpoint and a backend endpoint are accessible publicly, using HTTPS.

Depends on #4.

**Comment by strazdinsg** [comment]
Let's postpone the HTTPS, it is port of #48
